src/components/AmountTextInput.js

```diff
@@ -44,7 +52,7 @@ function AmountTextInput(props) {
             disableKeyboard
             autoGrow
             hideFocusedState
-            inputStyle={[styles.iouAmountTextInput, styles.p0, styles.noLeftBorderRadius, styles.noRightBorderRadius]}
+            inputStyle={[styles.iouAmountTextInput, styles.p0, styles.noLeftBorderRadius, styles.noRightBorderRadius, props.style]}
```

Comment:
I think we could do with using `...StyleUtils.parseStyleAsArray(props.style)` here as we're accepting both objects and array of objects. Same pattern for other instances like `containerStyles`.

Reply:
@perunt Just noticed that this causes a crash if you try to make a new money request:

![image](https://github.com/Expensify/App/assets/27287420/0636b0e9-45bb-4ed8-b74f-e22449e9374b)

Reply:
👍🏻